libcxx/include/__utility/integer_sequence.h

```diff
@@ -67,6 +72,30 @@ _LIBCPP_HIDE_FROM_ABI constexpr void __for_each_index_sequence(index_sequence<_I
 }
 #    endif // _LIBCPP_STD_VER >= 20
 
+#    if _LIBCPP_STD_VER >= 26
+// structured binding support for integer_sequence
+template <typename _Tp, _Tp... _Indices>
```

Comment:
I think we conventionally introduce type template parameters with `class` in produce code, as the standard wording consistently uses it in in library part. Ditto below.

```suggestion
template <class _Tp, _Tp... _Indices>
```

Reply:
AFAIK. We tend to try to keep to the standard wording as close as possible.